scripts/create_perf_json.py

```diff
@@ -1673,11 +1761,17 @@ def __init__(self, base_path: str):
             if 'atom' in files[shortname]:
                 files[shortname][
                     'e-core tma metrics'] = base_path + '/E-core_TMA_Metrics.csv'
-            cpu_metrics_url = f'{base_path}/{shortname}/metrics/perf/{shortname.lower()}_metric_perf.json'
+            cpu_metrics_url = f'{base_path}/{shortname}/metrics/perf/'
+            if longname[-1] == 'x':
```

Comment:
I'll double check with Caleb, but hopefully this extra logic won't be necessary soon. As an example, the file name https://github.com/captain5050/perfmon/blob/main/BDX/metrics/perf/broadwell_metrics_perf.json would be applicable for Broadwell client and should be renamed to `broadwellx_metrics_perf.json`.